lib/os/fdtable.c

```diff
@@ -23,8 +24,9 @@
 #include <zephyr/sys/speculation.h>
 #include <zephyr/internal/syscall_handler.h>
 #include <zephyr/sys/atomic.h>
+#include <zephyr/fs/fs.h>
 
-struct stat;
+K_MEM_SLAB_DEFINE(file_desc_slab, sizeof(struct fs_file_t), CONFIG_ZVFS_OPEN_MAX, 4);
```

Comment:
Note that after the commit 1cfdf7c8d1c4b0a5e5a88d5b5c01ce7fb84152c9, you need to use `ZVFS_OPEN_SIZE` define instead of `CONFIG_ZVFS_OPEN_MAX` (as the latter is set to 0 in typical use cases).